packages/extensions/openapi-to-typespec/test/arm-authorization/tsp-output/DenyAssignment.tsp

```diff
@@ -0,0 +1,62 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.Http;
+
+namespace Azure.ResourceManager.Authorization;
+@doc("Deny Assignment")
+model DenyAssignment is ExtensionResource<DenyAssignmentProperties> {
+  @doc("The ID of the deny assignment to get.")
+  @key("denyAssignmentId")
+  @segment("denyAssignments")
+  name: string;
+}
+
+@armResourceOperations
+interface DenyAssignments {
+  @doc("Get the specified deny assignment.")
+  get is ArmResourceRead<DenyAssignment>;
+  @doc("Gets deny assignments for a resource.")
+  listForResource is ArmResourceListByParent<
+    DenyAssignment,
+    {
+      ...BaseParameters<DenyAssignment>;
+
+      @doc("The ID of the target subscription.")
+      @minLength(1)
+      @path
+      subscriptionId: string;
+
+      @doc("The name of the resource group. The name is case insensitive.")
+      @maxLength(90)
+      @minLength(1)
+      @path
+      resourceGroupName: string;
+
+      @doc("The namespace of the resource provider.")
+      @path
+      resourceProviderNamespace: string;
+
+      @doc("The parent resource identity.")
+      @path
+      parentResourcePath: string;
+
+      @doc("The resource type of the resource.")
+      @path
+      resourceType: string;
+
+      @doc("The name of the resource to get deny assignments for.")
+      @path
+      resourceName: string;
+
+      @doc("The filter to apply on the operation. Use $filter=atScope() to return all deny assignments at or above the scope. Use $filter=denyAssignmentName eq '{name}' to search deny assignments by name at specified scope. Use $filter=principalId eq '{id}' to return all deny assignments at, above and below the scope for the specified principal. Use $filter=gdprExportPrincipalId eq '{id}' to return all deny assignments at, above and below the scope for the specified principal. This filter is different from the principalId filter as it returns not only those deny assignments that contain the specified principal is the Principals list but also those deny assignments that contain the specified principal is the ExcludePrincipals list. Additionally, when gdprExportPrincipalId filter is used, only the deny assignment name and description properties are returned.")
```

Comment:
What the orginal Swagger look like for this description?
Do we support the below in converter?
@doc("""
line1
line2
""")

Reply:
refer: https://github.com/Azure/azure-rest-api-specs/blob/a436672b07fb1fe276c203b086b3f0e0d0c4aa24/specification/authorization/resource-manager/Microsoft.Authorization/stable/2022-04-01/authorization-DenyAssignmentCalls.json#L504